pkg/k8s/namespaces/namespace.go

```diff
@@ -103,6 +104,13 @@ func (n *Namespaces) DestroyWithLabel(ctx context.Context, ns string, opts Delet
 			return err
 		}
 		deleteOpts := metav1.DeleteOptions{}
+
+		// It seems that by default, client-go don't delete pods scheduled by jobs, so we need to set the propation policy
+		if gvk.Kind == jobKind {
+			deletePropagation := metav1.DeletePropagationBackground
+			deleteOpts.PropagationPolicy = &deletePropagation
+		}
```

Comment:
@pchico83 with the example provided by Tere in the issue, the pod generated by a Job is not being deleted. As it is created by the job controller, it does not have the `deployed-by` label, and it seems that by default, client-go don't include the propagation policy (while `kubectl` does). So I'm including it in case of jobs (I see we already have this propagation policy on [our code](https://github.com/okteto/okteto/blob/master/pkg/k8s/jobs/crud.go#L60)). For now I just added it for Jobs, should I include it for other Kinds?

Reply:
@ifbyol nothing else comes to my mind